example/points/Makefile.am

```diff
@@ -7,12 +7,19 @@ if P4EST_ENABLE_BUILD_2D
 bin_PROGRAMS += example/points/p4est_points
 example_points_p4est_points_SOURCES = example/points/points2.c
 
+bin_PROGRAMS += example/points/p4est_generate_points
```

Comment:
Thanks; what about renaming to p4est_points_generate?

Reply:
ok will do.